packages/backend-core/src/context/mainContext.ts

```diff
@@ -157,6 +157,27 @@ export async function doInTenant<T>(
   return newContext(updates, task)
 }
 
+export async function doInSelfHostTenantUsingCloud<T>(
```

Comment:
I'd probably add a comment on this context function, the name does explain it, but just covers this whole area of functions explaining that self host licenses can connect to the cloud to use certain cloud services.